README.md

```diff
@@ -145,7 +145,7 @@ email in fitting places within your application.
 defmodule MyApp.SomeControllerPerhaps do
   def send_welcome_email do
     Email.welcome_email()   # Create your email
-    |> Mailer.deliver_now() # Send your email
+    |> Mailer.deliver_now!() # Send your email
```

Comment:
Do you think we should include documentation of `deliver_now` as well in the README?

Reply:
Yeah, maybe it'd be good to add a little more docs in the README about using `deliver_now` vs `deliver_now!`. I'll try to take a stab at that.